src/System.Runtime/tests/System/Guid.cs

```diff
@@ -5,7 +5,6 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-
```

Comment:
Just a nit, but the space after the system usings is preferred, unless I missed a change to that.